Geometry/RPCGeometryBuilder/src/RPCGeometryBuilder.cc

```diff
@@ -4,55 +4,47 @@
  Description: RPC Geometry builder from DD & DD4hep
               DD4hep part added to the original old file (DD version) made by M. Maggi (INFN Bari)
 //
-// Author:  Sergio Lo Meo (sergio.lo.meo@cern.ch) following what Ianna Osburne made for DTs (DD4HEP migration)
+// Author:  Sergio Lo Meo (sergio.lo.meo@cern.ch) following what Ianna Osborne made for DTs (DD4HEP migration)
 //          Created:  Fri, 20 Sep 2019 
+//          Modified: Fri, 29 May 2020, following what Sunanda Banerjee made in PR #29842 PR #29943 and Ianna Osborne in PR #29954    
 */
-#include "Geometry/RPCGeometryBuilder/src/RPCGeometryBuilderFromDDD.h"
+#include "Geometry/RPCGeometryBuilder/src/RPCGeometryBuilder.h"
 #include "Geometry/RPCGeometry/interface/RPCGeometry.h"
 #include "Geometry/RPCGeometry/interface/RPCRollSpecs.h"
-
 #include <DetectorDescription/Core/interface/DDFilter.h>
 #include <DetectorDescription/Core/interface/DDFilteredView.h>
 #include <DetectorDescription/DDCMS/interface/DDFilteredView.h>
 #include <DetectorDescription/DDCMS/interface/DDCompactView.h>
 #include <DetectorDescription/Core/interface/DDSolid.h>
-
-#include "Geometry/MuonNumbering/interface/DD4hep_MuonNumbering.h"
 #include "Geometry/MuonNumbering/interface/MuonGeometryNumbering.h"
 #include "Geometry/MuonNumbering/interface/MuonBaseNumber.h"
 #include "Geometry/MuonNumbering/interface/RPCNumberingScheme.h"
 #include "Geometry/CommonTopologies/interface/TrapezoidalStripTopology.h"
-
 #include "DataFormats/GeometrySurface/interface/RectangularPlaneBounds.h"
 #include "DataFormats/GeometrySurface/interface/TrapezoidalPlaneBounds.h"
-
 #include "DataFormats/GeometryVector/interface/Basic3DVector.h"
-
 #include <iostream>
 #include <algorithm>
 #include "DetectorDescription/DDCMS/interface/DDSpecParRegistry.h"
-#include "Geometry/MuonNumbering/interface/DD4hep_RPCNumberingScheme.h"
-
 #include "DataFormats/Math/interface/CMSUnits.h"
 #include "DataFormats/Math/interface/GeantUnits.h"
 
 using namespace cms_units::operators;
 
-RPCGeometryBuilderFromDDD::RPCGeometryBuilderFromDDD(bool comp11) : theComp11Flag(comp11) {}
+RPCGeometryBuilder::RPCGeometryBuilder(bool comp11) : theComp11Flag(comp11) {}
 
-RPCGeometryBuilderFromDDD::~RPCGeometryBuilderFromDDD() {}
+RPCGeometryBuilder::~RPCGeometryBuilder() {}
```

Comment:
in C++11 and above you do not need to write an empty destructor. Please, remove it.

Reply:
done